backend/apis/health.api.js

```diff
@@ -0,0 +1,5 @@
+const { app } = require("../server");
+
+app.get("/health", (req, res) => {
```

Comment:
I'd actually prefer this to be under `/api/health` if that's possible to configure EB for, since the way our web server works is that everything not under `/api` returns the HTML file.

Reply:
Okay, we can do it this way. Or we can just pass the health check on the 302 redirect. What do you think is going to be the best option?

Reply:
My guess is that EB's health check looks for a http 200 status and that 302 is just not okay at all (it doesn't care about response body in that case).

If we can't change EB to look for `/api/health` instead of `/health`, then it's okay for us to merge this in as is. It just means the frontend can never have a route at `/health`